examples/src/main/scala/spire/example/simplification.scala

```diff
@@ -1,12 +1,19 @@
 package spire
 package example
 
+object Simplification {
```

Comment:
Some leftover code here?

Reply:
this example doesn't compile right now, due to incompatability around `newBuilder` method - you *have* to override pre-2.13, but post 2.13 the `override` is a compile errror

Reply:
I don't see why this example needs to connect with the scala collection hierarchy. It's being used internally in one of the example cases; so let's just copy/paste the collection methods that are required for the example to work and move on.

(I don't understand what ties the enumeration functions and the simplification part; and I don't recognize the simplification algorithm. Does it correspond to a continued fraction expansion?)

Reply:
There is a surprising (to me) amount of Builder, SeqLike, etc, code in several of the examples.  I didn't expend much time thinking about it but it seemed a bit over-designed.

Reply:
Do not hesitate to open an issue specifying which files should be simplified. That said, I'm not very familiar with the current examples -- I just noticed that many apply to quite abstract mathematical problems.